docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading pod logs using the CLI
+[role="_abstract"]
+
+
+Each pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. You can use the CLI to download pod logs.
+
+.Procedure
+
+. Obtain the pod names:
++
+----
+$ oc get pods
+----
++
+The output resembles the following:
++
+----
+NAME                       READY     STATUS      RESTARTS   AGE
+eap-builder-1-build        0/1       Completed   0          1d
+mta-postgresql-1-hfbdn     1/1       Running     0          1d
+mta-sso-1-build            0/1       Completed   0          1d
+mta-web-console-1-build    0/1       Completed   0          1d
+mta-web-console-1-vt7s5    1/1       Running     1          1d
+sso-1-wjl2n                1/1       Running     1          1d
+----
+
+. Use `oc logs` to examine the pod log.
++
+----
+$ oc logs <POD_NAME>
+----
+
+. The output may be redirected to obtain a copy of the current log:
```

Comment:
This step is not worded like a step. Since it is an option, I think it should be added to the previous step. Could be a note.